src/work-allocation/components/derived-icon-field/derived-icon-field.component.html

```diff
@@ -0,0 +1,5 @@
+<svg *ngIf="task&&sourceColumn&&task[sourceColumn]==matchValue" class="hmcts-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25" path="" style="fill: orange">
+    <title>This case has more than one task. Review needed.</title>
+    <path d="M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8
+    C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z"></path>
+</svg>
```

Comment:
I think this may fail linting as it needs a line break at the end.

Reply:
Added